### PR TITLE
docs(website): add new guide for git

### PR DESCRIPTION
**Pull request by jakepage91**
## 📑 Description
Added Git guide to guides section

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

## ℹ Additional Information
I also added a GitHub star button

### CODE REVIEW

File: website/guides/03_git-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,425 @@
+---
+title: The guide to Git I never had.
+slug: git
+description: 'Install ingress-nginx for Kubernetes with Glasskube package manager including a GUI, CLI and automatic package updates'
```
Comment (pmig) [inline website/guides/03_git-guide.mdx:4]: This description is important for Google. Maybe we can highlight the inidvidual commands that are explained - The description should not exceed 160 chars.

**Review by pmig — CHANGES_REQUESTED** [review]
Love the integration of the star Button, will reuse it as soon as this PR is merged

File: website/guides/03_git-guide.mdx (hunk 1 above)
Comment (jakepage91) [reply website/guides/03_git-guide.mdx]: Done!

File: website/sidebar-guides.ts [hunk 2]
```diff
@@ -14,7 +14,19 @@ const sidebars: SidebarsConfig = {
...
+        description:
+          '⚠️ Contributors are what make open source great, here is where we share some resources to support you. ⚠️',
+        slug: '/categories/contributors',
+      },
+      items: ['git-guide'],
+    },
   ],
 };
 
-export default sidebars;
```
Comment (pmig) [inline website/sidebar-guides.ts:20]: @jakepage91 these empty lines should stay :-)

**Review by pmig — COMMENTED** [review]
We prefer to end files with empty lines.

**Review by pmig — APPROVED** [review]

**Pull request merged**